frontend/components/ClubEditPage/ClubEditCard.tsx

```diff
@@ -397,6 +399,57 @@ export default function ClubEditCard({
     {
       name: 'General',
       type: 'group',
+      description: (
+        <div className="mb-4">
+          <a onClick={() => setShowRankModal(true)} className="is-italic">
+            How can filling out this information help your club's ranking?
```

Comment:
Is there a reason we're italicizing? I can't remember doing it anywhere else on the frontend.

Reply:
Taking another look at this: we should probably make this less overt, since OSA will see this whenever they open the club management view. Let's change this to something like "How does filling out this information affect your club's page?" -- anything that doesn't involve the word "rank".